src/el/el_client_context.star

```diff
@@ -7,6 +7,7 @@ def new_el_client_context(
 	ws_port_num,
 	engine_rpc_port_num,
 	jwt_secret,
+	el_metrics_info,
```

Comment:
can this be the absolute last thing and default it to None `el_metrics_info=None`